trunk/SUAVE/Methods/Aerodynamics/AVL/write_geometry.py

```diff
@@ -189,15 +205,44 @@ def make_section_text(avl_section):
 
     return section_text
 
+    
+def make_body_section_text(avl_body_section):
+    # Template for a section
+    section_base = \
+'''
+SECTION
+#Xle     Yle      Zle      Chord     Ainc  Nspanwise  Sspace
+{0}    {1}     {2}     {3}     {4}      1        0
+'''
+    airfoil_base = \
+'''
+AFILE
+{}
+'''
 
+    # Unpack inputs
+    x_le    = avl_body_section.origin[0]
+    y_le    = avl_body_section.origin[1]
+    z_le    = avl_body_section.origin[2]
+    chord   = avl_body_section.chord
+    ainc    = avl_body_section.twist
+    airfoil = avl_body_section.airfoil_coord_file
 
+    section_text = section_base.format(x_le,y_le,z_le,chord,ainc)
+    if airfoil:
+        section_text = section_text + airfoil_base.format(airfoil)
+    for cs in avl_body_section.control_surfaces:
+        control_text = make_controls_text(cs)
+        section_text = section_text + control_text
+
+    return section_text
+
+    
 def make_controls_text(avl_control_surface):
```

Comment:
No this has not. I do not recall seeing Fidelity 0 use flaps so I decided to not edit any of this code